Kitodo/src/main/java/org/kitodo/services/data/ProcessService.java

```diff
@@ -928,10 +928,8 @@ public Fileformat readMetadataFile(Process process) throws ReadException, IOExce
         }
 
         Fileformat ff = determineFileFormat(type, process);
-
         try {
-            // TODO: this is not working anymore because uri is not full path
-            ff.read(metadataFileUri.toString());
+            ff.read(new File(metadataFileUri).toString());
```

Comment:
It is not about the general working or not more a thing for watching of leaking resources. If this call is done per day 1.000 ore more times then we got maybe some got real problems about non-collectable resources.